fix(curriculum): remove media query requirement, general improvements (#46639)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/14-responsive-web-design-22/learn-css-flexbox-by-building-a-photo-gallery/6153938dce8b294ff8f5a4e9.md

The `justify-content` property determines how the items inside a flex container are positioned along the main axis, affecting their position and the space around them.

> Give your `.gallery` selector a `justify-content` property set to `center` as the value.

Reviewer: ```suggestion
Give your `.gallery` selector a `justify-content` property of `center`.
```

Author: End of sentence code block.

Reviewer: updated

body {
  margin: 0;
  font-family: sans-serif;
  background: #f5f6f7;
}

.header {
  text-align: center;
  text-transform: uppercase;
  padding: 32px;
  background-color: #0a0a23;
  color: #fff;
  border-bottom: 4px solid #fdb347;
}

--fcc-editable-region--
.gallery {
  display: flex;

>   flex-direction: row;

Reviewer 2: ```suggestion
  flex-direction: row-reverse;
```
